boolean/nand/a5f368aa-2cea-4110-a170-c08ec18a6a63.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "boolean/nand",
+  "description": "Performs a boolean NAND operation on the values received via inputs `a` and `b`.\n\nExample:\n\n1. true@0 is received via `a`\n2. true@0 is received via `b`\n3. false@0 is sent via `nand`",
+  "interface": {
+    "inputs": {
+      "ad9e2497-ee64-46bc-ae85-86ce71a33bd9": {
```

Comment:
Please add some short description to input and output ports also to help users trying to integrate the node into their project.

Reply:
descriptions added as suggested